src/help/h_tags.php

```diff
@@ -51,18 +48,17 @@ function print_finding() {
     '</div></div></div>',
     "</div></div></div></div><hr class=\"c\">
 
-<p>To find all papers with tag “#discuss”:</p>
+<p>To find all applications with tag “#discuss”:</p>
 
 <div class=\"mb-p\">", $hth->search_form("#discuss"), "</div>
 
-<p>You can also search with “", $hth->search_link("show:tags"), "” to see each
-paper’s tags, or “", $hth->search_link("show:#tagname"), "” to see a particular tag
+<p>You can also search with “", $hth->search_link("show:tags"), "” to see each application’s tags, or “", $hth->search_link("show:#tagname"), "” to see a particular tag
```

Comment:
Could you not change the line breaks but just the text? again easier for future merge.